compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -1130,10 +1132,159 @@ int64_t getMinElementBitwidth(linalg::LinalgOp linalgOp) {
   return bitwidth;
 };
 
+//===---------------------------------------------------------------------===//
+// Bufferization utility functions
+//===---------------------------------------------------------------------===//
+
+/// Get strides for row-major oredering of a tensor with the given `shape`.
+static SmallVector<int64_t> getStridesFromShape(ArrayRef<int64_t> shape) {
+  if (shape.empty()) {
+    return {};
+  }
+  SmallVector<int64_t> strides(shape.size(), ShapedType::kDynamic);
+  strides.back() = 1;
+  for (int i = strides.size() - 1; i > 0; --i) {
+    if (ShapedType::isDynamic(shape[i])) {
+      break;
+    }
+    strides[i - 1] = strides[i] * shape[i];
+  }
+  return strides;
+}
+
+static MemRefType
+getMemrefTypeForTensor(IREE::TensorExt::DispatchTensorType tensorType,
+                       MemRefLayoutAttrInterface layout = {},
+                       Attribute memorySpace = {}) {
+  return MemRefType::get(tensorType.getShape(),
+                         tensorType.getBoundElementType(), layout, memorySpace);
+}
```

Comment:
This function is only used once, and the code is simple. Do we delete this method and replace the use with `MemRefType::get(...)` directly?